packages/block-library/src/image/view.js

```diff
@@ -17,6 +17,43 @@ const focusableSelectors = [
 	'[tabindex]:not([tabindex^="-"])',
 ];
 
+// I've defined the scroll handling outside of the store
+// to prevent confusion as to how this logic is called.
+// This logic is not referenced directly by the directives
+// in the markup because the scroll event we need to listen
+// to is triggered on the window; so we define it outside of
+// the store to signal that the behavior here is different.
+// If we find a compelling reason to move it to the store,
+// feel free to do so.
```

Comment:
if we turn these into JSDoc comments then the explanation will follow the variables and functions. otherwise they are only evident to those who look at this spot where they're created.

```js
/**
 * Stores a context-bound scroll handler.
 *
 * This callback could be defined inline inside of the store
 * object but it's created externally to avoid confusion about
 * it's logic…
 *
 * @var {function}
 */
let scrollCallback;

/** 
 * Tracks whether user is touching screen; used to differentiate mobile and desktop behaviors.
 *
 * @var {boolean}
 */
let isTouching = false;

/** 
 * Used to differentiate mobile and desktop behaviors.
 *
 * @var {boolean}
 */
let isTouching = false;

/**
 * Lightbox page-scroll handler: prevents scrolling.
 *
 * This handler is added to prevent scrolling behaviors that
 * trigger content shift while the lightbox is open…
 *
 * It would be better to accomplish this through CSS alone, but
 * because … a JavaScript version is required to avoid … in
 * order to provide the best visual experience.
 *
 * @param {object} context Interactivity page context?
 */
function handleScroll( context ) {
	…
}
```

Reply:
Thanks for the heads up! I've converted these all to JSDoc comments.